src/main/java/race/util/RandomNumberUtil.java

```diff
@@ -0,0 +1,12 @@
+package race.util;
+import java.util.Random;
+
+public class RandomNumberUtil {
+
+    private static final Random RANDOM = new Random();
+    public static final int BOUND = 10;
```

Comment:
```suggestion
    private static final int BOUND = 10;
```

public으로 공개하지 않아도 될 것 같아요